Thirtys/Screens/StreakScreen.swift

```diff
@@ -36,7 +35,7 @@ struct StreakScreen: View {
                             }
                         }
                         
-                        Text("Congratulations on your 16-day streak! 🎉 Keep up the great work—consistency is key to success. Just a 30 minutes each day can make a huge difference! 🚀")
+                        Text("Congratulations on your \(vm.streaks.count)-day streak! 🎉 Keep up the great work—consistency is key to success. Just a 30 minutes each day can make a huge difference! 🚀")
```

Comment:
tolong buat untuk descriptionnya kalau ada 2 state
1. streaknya masih 0 -> ("Let's start your learning journey today! Every great achievement begins with the first step.")
2. streaknya tadinya 10 jadi 0 -> ("You did it before, and you can do it again. Begin a new streak and keep pushing forward!")